docs/user-guide/configure-zowe-zosmf-workflow.md

```diff
@@ -99,7 +99,7 @@ The **Create configuration** step is mandatory. A valid zowe.yaml is required to
 
 This step consumes the zowe.yaml configuration file you created in the previous step. It contains three sub-steps:
 
-1. **(Optional) Run the Zowe install (only for convenience build)**. This step is required only for the convenience build. It runs the `zwe install` command.
+1. **Run the Zowe install**. This step is required only for the convenience build. It runs the `zwe install` command.
```

Comment:
Is removing this language that specified the Convenience build option adequate to resolve this issue?

Reply:
No, you are right, remove also second sentence. _This step is required only for the convenience build._